routes/projectComments.js

```diff
@@ -13,6 +236,45 @@ router.post(
   asyncHandler(createComment)
 );
 
+/**
+ * @swagger
+ * /projects/{projectId}/comments:
+ *   get:
+ *     summary: Get comments for a project
+ *     description: Retrieves a list of paginated comments for a specific project
+ *     tags: [Comments]
+ *     parameters:
+ *       - in: path
+ *         name: projectId
+ *         required: true
+ *         schema:
+ *           type: string
+ *           format: uuid
+ *         description: Project ID
+ *         example: "987fcdeb-51a2-43d1-9c4f-123456789abc"
+ *        - $ref: '#/components/parameters/limitParam'
+ *        - $ref: '#/components/parameters/offsetParam'
+ *     responses:
+ *       200:
+ *         description: Comments retrieved successfully
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: '#/components/schemas/CommentListResponse'
+ *       400:
+ *         description: Invalid project ID format
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: '#/components/schemas/ValidationErrorResponse'
+ *       500:
+ *         description: Server error
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: '#/components/schemas/InternalErrorResponse'
```

Comment:
This is currently causing an error in the console, as such your GET project comments documentation isn't displaying.